index.html

```diff
@@ -150,9 +150,13 @@
             margin: 0 auto;
         }
 
-        .downloadexplain {
+        .downloadarea {
             color: #fff;
-            font-size: 30px;
+        }
+
+        .linebrBefore:before {
```

Comment:
What's the purpose of this class? Could this be solved using padding or margin instead?